runner.py

```diff
@@ -54,6 +55,14 @@ def run_agent(terminate_event, callback_event, config_file, name, team, index, m
 
 
 def main():
+    # Inject DLL
+    injector_dir = os.path.join(os.path.dirname(__file__), "RLBot_Injector.exe")
+    incode=subprocess.call([injector_dir, 'hidden'])
+    injector_codes=['INJECTION_SUCCESSFUL','INJECTION_FAILED','MULTIPLE_ROCKET_LEAGUE_PROCESSES_FOUND','RLBOT_DLL_ALREADY_INJECTED','RLBOT_DLL_NOT_FOUND','MULTIPLE_RLBOT_DLL_FILES_FOUND']
+    print(injector_codes[incode])
+    if incode!=0 or incode!=3:
```

Comment:
If statement will always be true.
incode = 0 would be -> False or True -> True -> exit()

It would be nicer to compare to the strings instead of index too. Ex. not equal to INJECTION_SUCCESSFUL and not equal to RLBOT_DLL_ALREADY_INJECTED.